src/content/docs/infrastructure/new-relic-infrastructure/troubleshooting/network-connection-issues.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: Network connectivity issues
+type: troubleshooting
+tags:
+  - Infrastructure
+  - Infrastructure monitoring troubleshooting
+  - Troubleshoot logs
+  - Network troubleshooting
+metaDescription: Troubleshooting suggestions if the New Relic infrastructure agent is not able to connect to the New Relic endpoints
+---
+
+## Problem
+
+The infrastructure agent is experiencing network issues connecting to New Relic endpoints.
+Connectivity errors such as "Collector endpoint not reachable" or "Timeout exceeded while awaiting headers" appear on the agent logs.
+
+## Solution
+
+Verify that the New Relic infrastructure agent endpoints are reachable from the host where it's installed.
+Follow the instructions below to confirm there are no issues with your environment's network settings:
+
+<CollapserGroup>
+  
+<Collapser id="windows" title="Windows">
+
+Run the following command on the terminal:
+```
+$ Invoke-WebRequest -Uri "https://infra-api.newrelic.com/cdn-cgi/trace"
```

Comment:
```suggestion
Invoke-WebRequest -Uri "https://infra-api.newrelic.com/cdn-cgi/trace"
```

Reply:
I removed all the $ signs so it doesn't interrupt the user flow of copying/pasting the code from the doc.